landing-page/content/common/spark-quickstart.md

```diff
@@ -0,0 +1,370 @@
+  TripID bigint,
+  Trip_distance float,
+  Fare_amount double,
+  Store_and_fwd_flag string
+)
+USING iceberg;
+```
+{{% /tabcontent %}}
+{{% tabcontent "SparkShell" %}}
+```scala
+import org.apache.spark.sql.types.{DoubleType, FloatType, LongType, StringType, StructField, StructType}
+import org.apache.spark.sql.Row
+val schema = StructType( Array(
+    StructField("VendorID", LongType,true),
+    StructField("TripID", LongType,true),
+    StructField("Trip_distance", FloatType,true),
+    StructField("Fare_amount", DoubleType,true),
+    StructField("Store_and_fwd_flag", StringType,true)
+))
+val df = spark.createDataFrame(spark.sparkContext.emptyRDD[Row],schema)
+df.writeTo("prod.nyc.taxis").create()
+```
+{{% /tabcontent %}}
+{{% tabcontent "PySpark" %}}
+```py
+from pyspark.sql.types import DoubleType, FloatType, LongType, StructType,StructField, StringType
+schema = StructType([
+  StructField("VendorID", LongType(), True),
+  StructField("TripID", LongType(), True),
+  StructField("Trip_distance", FloatType(), True),
+  StructField("Fare_amount', DoubleType(), True),
+  StructField("Store_and_fwd_flag', StringType(), True)
+])
+
+df = spark.createDataFrame([], schema)
+df.writeTo("prod.nyc.taxis").create()
+```
+{{% /tabcontent %}}
+{{% /codetabs %}}
+
+Iceberg catalogs support the full range of SQL DDL commands, including:
+
+* [`CREATE TABLE ... PARTITIONED BY`](../spark-ddl#create-table)
+* [`CREATE TABLE ... AS SELECT`](../spark-ddl#create-table--as-select)
+* [`ALTER TABLE`](../spark-ddl#alter-table)
+* [`DROP TABLE`](../spark-ddl#drop-table)
+
+### Writing
+
+Once your table is created, insert data using [`INSERT INTO`](../spark-writes#insert-into):
+
+{{% codetabs "InsertData" %}}
+{{% addtab "SparkSQL" checked %}}
+{{% addtab "SparkShell" %}}
+{{% addtab "PySpark" %}}
+{{% tabcontent "SparkSQL"  %}}
+```sql
+INSERT INTO prod.nyc.taxis
+VALUES (1, 1000371, 1.8, 15.32, 'N'), (2, 1000372, 2.5, 22.15, 'N'), (2, 1000373, 0.9, 9.01, 'N'), (1, 1000374, 8.4, 42.13, 'Y');
+```
+{{% /tabcontent %}}
+{{% tabcontent "SparkShell" %}}
+```scala
+import org.apache.spark.sql.Row
+
+val schema = spark.table("prod.nyc.taxis").schema
+val data = Seq(
+    Row(1: Long, 1000371: Long, 1.8f: Float, 15.32: Double, "N": String),
+    Row(2: Long, 1000372: Long, 2.5f: Float, 22.15: Double, "N": String),
+    Row(2: Long, 1000373: Long, 0.9f: Float, 9.01: Double, "N": String),
+    Row(1: Long, 1000374: Long, 8.4f: Float, 42.13: Double, "Y": String)
+)
+val df = spark.createDataFrame(spark.sparkContext.parallelize(data), schema)
+df.writeTo("prod.nyc.taxis").append()
+```
+{{% /tabcontent %}}
+{{% tabcontent "PySpark" %}}
+```py
+schema = spark.table("prod.nyc.taxis").schema
+data = [
+    (1, 1000371, 1.8, 15.32, "N"),
+    (2, 1000372, 2.5, 22.15, "N"),
+    (2, 1000373, 0.9, 9.01, "N"),
+    (1, 1000374, 8.4, 42.13, "Y")
+  ]
+df = spark.createDataFrame(data, schema)
+df.writeTo("prod.nyc.taxis").append()
+```
+{{% /tabcontent %}}
+{{% /codetabs %}}
+
```

Comment:
Should there be a section presenting how the table data & metadata are persisted on the disk or is this too low-level for this scope?

Reply:
I think that might be too low-level. IMO the quickstart should be short and easy to run, and then forward the reader into the docs where they can choose the next details they want to dive into.